hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/FederationUtil.java

```diff
@@ -98,13 +98,10 @@ public static JSONArray getJmx(String beanQuery, String webAddress,
       JSONObject json = new JSONObject(jmxOutput);
       ret = json.getJSONArray("beans");
     } catch (IOException e) {
-      LOG.error("Cannot read JMX bean {} from server {}: {}",
-          beanQuery, webAddress, e.getMessage());
-    } catch (JSONException e) {
```

Comment:
I would keep this one separately as we shouldn't need more details on this if the JSON parsing fails.